config/locales/en.yml

```diff
@@ -90,47 +90,47 @@ en:
         course:
           attributes:
             level:
-              blank: "^You need to pick a level"
+              blank: "^Select a level"
```

Comment:
What's the question that triggers this? Is it supposed to read 'Select an A level'? Or 'Select a level (of something)'?

Reply:
Level of education: Primary, Secondary or Further:

<img width="560" alt="Screenshot 2021-07-06 at 10 38 00" src="https://user-images.githubusercontent.com/813383/124578616-444e6f00-de46-11eb-90f5-3160b85af80b.png">

Reply:
Ok, so yeah, 'Select a course level'